app/models/proxy.rb

```diff
@@ -537,6 +537,7 @@ class PoliciesConfig
     include Enumerable
 
     delegate :each, :to_json, :as_json, to: :policies_config
+    alias to_s to_json
```

Comment:
This appears to me as a bug in enhanced adapter. Seems like `to_s` is called on the policies_config attribute instead of going through the `#serialize` method. I still have no idea why and how does that happen.

Without this alias, the database record is updated with a string like 
```
[5] pry(main)> proxy.attributes_before_type_cast
...
 "policies_config"=>
  "#<Proxy::PoliciesConfig:0x000000000bdb6590>\",\"configuration\":{},\"enabled\":false},{\"name\":\"apicast\",\"version\":\"builtin\",\"configuration\":{},\"enabled\":true}]"
...
```